Stats preview card component

Daniel4lm opened the pull request:
## Subject

This pull request contains the solution to the Stats preview card component challenge on Frontend Mentor, 
as a part of my Optimum BH Internship. 

The task was to build out preview company card component, which represents two sections. One section is details section , implemented mostly by using Flexbox CSS model. The other one is the header image slightly covered by  violet color.
On mobile layout those section where reversed, thanks to flex ability, so picture holds upper position instead of right on desktop layout.

### The solution on this branch contains the following:

- [x] Html page containing the corresponding DOM structure, including the Optimum header with anchor tags with urls(whose presense was requested by script tests).
- [x] Tailwind configuration file containing the extended theme atributes, like colors, sizes, breakpoints, etc.
- [x] Challenge starter files, incl. images and style guide
- [x] Solution design images(mobile & desktop layout)
- [x] CSS style file with tailwind directives and custom classes

## *Note*

After finishing the test, created screenshot of mobile version is represented in much smaller height so it hides the lower part of he card.

Daniel4lm committed:
Initial commit

- Include challenge starter files
- Add extended tailwind styling

Daniel4lm committed:
- Include more preview card styling

Daniel4lm committed:
- Complete html and css styling according to style guide
- Change in tailwind configuration file

Daniel4lm committed:
- Include readme.md file
- Place design images inside challenge folder

Daniel4lm committed:
Merge branch 'optimumBA:stats_preview_card_component' into stats_preview_card_component

Daniel4lm committed:
Modify readme.md

Daniel4lm committed:
Fix image path

Daniel4lm committed:
Minor css fixes

Daniel4lm committed:
Merge branch 'optimumBA:stats_preview_card_component' into stats_preview_card_component

Daniel4lm committed:
Revert back some default styles

almirsarajcic commented on challenges/stats_preview_card_component/README.md:
```diff
@@ -0,0 +1,125 @@
+@tailwind utilities;
+@layer components {
+  /* custom css class */
+  .qr-card {
```
Since this `README.md` file is not required I'd recommend you delete it so that you don't have to maintain it. You can leave it if you want, but in that case, remove irrelevant parts.

Daniel4lm replied:
> Since this `README.md` file is not required I'd recommend you delete it so that you don't have to maintain it. You can leave it if you want, but in that case, remove irrelevant parts.

Sure, I'll remove the from all branches

almirsarajcic commented on challenges/index.html:
```diff
@@ -51,11 +53,14 @@
       </div>
 
       <div class="mt-3">
+        <p>QR code component -</p>
```
Same as in the other PR, change this file only in `main` branch.

almirsarajcic commented on challenges/stats_preview_card_component/style-guide.md:
```diff
@@ -0,0 +1,36 @@
+# Front-end Style Guide
```
You can remove this file.

almirsarajcic commented on challenges/stats_preview_card_component/index.html:
```diff
@@ -25,39 +25,124 @@
+    <!-- Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lexend+Deca&display=swap"
+      rel="stylesheet"
+    />
```
Use one URL for multiple fonts to reduce the number of HTTP requests.

Daniel4lm replied:
I see the point, thanks

Daniel4lm replied:
> Use one URL for multiple fonts to reduce the number of HTTP requests.

I see know the point, thanks

almirsarajcic reviewed (changes requested):
The mobile variant looks ok, but I can't scroll down to see the whole card. The screenshot reflects it. Also, in the desktop variant credits overlap the card on my screen.

Daniel4lm committed:
Update component

- Remove README.md & style-guide.ms files
- Reduce google font request into one inside head section
- Make credits visible in both layouts
- Make card scrollable on mobile variant

almirsarajcic reviewed (approved):
Looks great! 👏
